client/sites/settings/sftp-ssh/sftp-form.tsx

```diff
@@ -413,13 +413,7 @@ export const SftpForm = ( { disabled }: SftpFormProps ) => {
 	}
 
 	return (
-		<ContainerComponent
-			className="sftp-card"
-			headingId="sftp-credentials"
-			title={
-				siteHasSshFeature ? translate( 'SFTP/SSH credentials' ) : translate( 'SFTP credentials' )
-			}
```

Comment:
how come this is no longer necessary? Feels like a wholly separate issue 🤔

Reply:
This code is a bit strange, it resolves the component dynamically. Look at lines 399+ of this same file, the ContainerComponent possibilities don't share a common interface. Before these changes we were unit testing against the previous implementation, which has those props. Now, those props do not exist and tests fail because of it.